eblob_kit.py

```diff
@@ -521,6 +524,8 @@ def __len__(self):
 class Blob(object):
     """Abstraction to blob consisted from index and data files."""
 
+    CREATE_MODE = 'wb'
```

Comment:
I don't see any reason to have `Blob.CREATE_MODE`. It is used only in one method and also it means `OPEN_MODE`. Let's make it local for `create()`

Reply:
Removed.